### PR TITLE
Add rubygem-hammer_cli_sam

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: rubygem-hammer_cli_sam/rubygem-hammer_cli_sam.spec [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+URL:     http://github.com/Katello/hammer-cli-sam
+Source0: https://rubygems.org/downloads/%{gem_name}-%{version}.gem
+
+BuildArch: noarch
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+%if 0%{?scl:1}
+Obsoletes: rubygem-%{gem_name} < 1.0.1-2
+%endif
+
+%if 0%{?fedora} > 18
```
Comment (komidore64) [inline rubygem-hammer_cli_sam/rubygem-hammer_cli_sam.spec]: Is this Fedora conditional needed? Fedora 18 is ancient these days.

File: rubygem-hammer_cli_sam/rubygem-hammer_cli_sam.spec [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+Requires:  %{?scl_prefix}ruby(release)
+%else
+Requires:  %{?scl_prefix}ruby(abi)
+%endif
+Requires: %{?scl_prefix}ruby(rubygems)
+Requires: %{?scl_prefix}rubygem(hammer_cli) >= 0.1.1
```
Comment (komidore64) [inline rubygem-hammer_cli_sam/rubygem-hammer_cli_sam.spec]: I would think this needs to depend on `hammer_cli_katello`, not plain `hammer_cli`?

**Comment by ehelms** [comment]
@komidore64 updated

**Comment by komidore64** [comment]
looks p good.

ACK

**Pull request merged**